model-engine/model_engine_server/inference/batch_inference/generate_tool_sample_data.py

```diff
@@ -0,0 +1,79 @@
+import json
```

Comment:
just confirming that we want to add this file to `main` because we didn't include one for `sample_data.json`

Reply:
i used this script to avoid dealing with json escaping. sample_data.json is hand-crafted since its content is simple